setup.py

```diff
@@ -18,7 +19,14 @@
         'url-matcher',
         'tldextract',
         'sqlitedict',
+        'scrapy',
```

Comment:
I'm not quite sure what you mean by adding `scrapy-poet` since it's an idempotency in itself. For the `requirements.txt` however, I'll be adding that alongside a new pre-commit hook setup in another PR.